pfcpiface/pfcpiface.go

```diff
@@ -28,10 +28,13 @@ type sessRecord struct {
 }
 
 var sessions map[uint64]sessRecord
+var sequence_number uint32 = 1
```

Comment:
should this be random?

Reply:
No. Peer node uses Port + IP + Seq number to detect retransmission. So we should go in -order. This helps us to avoid sending 2 messages with same sequence number in short period of time.